### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by NataJenkins**
Unit Testing for To do application using Jest

### CODE REVIEW

**Review by umararshadbut — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Such a remarkable project you made :+1: ! Awesome README :100: !
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested changes  👇.
- [x] Remove the `console.log`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png) from Pathwright)
- [x] Please move the test directory into the root directory.
- [ ]  Please add the test instruction in the README file.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] Please add the `.vscode` into the `.gitignore`.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**Umair**
**Happy Coding :smiley:**

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @NataJenkins  :clap: 

# Status: Changes Required! ♻️ 

**Kindly attend to the previous reviewer request;**

**_Could it be from my end 🤔, I see no changes._**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/projects.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  anchorElement.textContent = name;
+  projectTitle.appendChild(anchorElement);
+  projectTitle.addEventListener("click", () => {
+    changeProjectName(name, id);
+  });
+  return projectTitle;
+};
+const getLocalStorageProjects = () => {
+  return JSON.parse(localStorage.getItem("projects"));
+  console.log(JSON.parse(localStorage.getItem("projects")));
```
Comment (bellom) [inline src/projects.js]: Kindly remove this line.

File: src/test/projects.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { addProject } from "../projects";
+
+it("works", () => {
+  expect(1).toEqual(1);
+});
+
+describe("Projects", () => {
+  const testProject = addProject({ name: "hola" });
+  console.log(testProject);
```
Comment (bellom) [inline src/test/projects.test.js]: Remove this line.

File: src/test/tasks.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+});
+
+describe("Tasks", () => {
+  const testTask = addTask({
+    title: "testTask",
+    description: "testDescription",
+    date: "03/03/2021",
+    priority: 1,
+  });
+  console.log(testTask);
```
Comment (bellom) [inline src/test/tasks.test.js]: Remove this line.

File: README.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+- JavaScript
+- Linters
+- VS code
+- Webpack
+
+## :construction_worker: Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page](https://github.com/pointerish/Books/).
```
Comment (bellom) [inline README.md]: Kindly add the test instruction in the README file(That is; how to run the test)

**Review by bellom — CHANGES_REQUESTED** [review]

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**